docs/docs-content/clusters/edge/edgeforge-workflow/prepare-user-data.md

```diff
@@ -45,9 +140,55 @@ Installer configuration file and the OS pack support the usage of cloud-init sta
    From the **CanvOS** directory, copy the **user-data.template** file and name the copy **user-data**. This is a
    template that you can use as a starting point to build your own user data file.
 
-### Configure Installation Mode
+2. View the available git tag.
+
+   ```shell
+   git tag
+   ```
+
+3. Check out the latest available tag. This guide uses the tag v4.6.c as an example.
+
+   ```bash
+   git checkout v4.6.c
+   ```
+
+### Prepare .arg File
+
+1. Specify the system architecture and OS distribution and version. These configurations will apply to both the OS of
+   your Edge host before and after cluster formation.
+
+2. Specify the Kubernetes distribution. The Kubernetes distribution is used together with the OS distribution and
+   version to create an immutable provider image that has your specified OS and Kubernetes.
+
+   The Kubernetes version is specified in a different file named `k8s_version.json`.
 
-2. Decide whether you want to deploy an Edge host that is managed locally or centrally by Palette. The default
+3. Specify the image registry, image repository name, and image tag that will be used to tag your provider images. The
+   custom tag, together with the Palette agent version (the same number as the Git tag you are using), the version and
+   distribution of Kubernetes and the OS distribution used by the image forms the tag of the image.
+
+   For example, if your `.arg` file contains the following arguments, the full image reference would be
+   `ttl.sh/ubuntu:k3s-1.32.1-v4.6.c-demo`.
+
+   ```text
+   IMAGE_REGISTRY=ttl.sh
+   IMAGE_REPO=ubuntu
+   CUSTOM_TAG=demo
+   K8S_DISTRIBUTION=k3s
+   OS_DISTRIBUTION=ubuntu
+   OS_VERSION=22
+   ```
+
+4. (Optional) If your build machine isn't in a restricted network environment, or your build process does not require
+   access to a proxy server, skip this step.
+
+   You can use `HTTP_PROXY` and `HTTPS_PROXY` to specify the URLs of the proxy servers to be used for your build.
+
+5. Refer to [Edge Artifact Build Configurations](./palette-canvos/arg.md) for a comprehensive list of arguments you can
+   use to customize the build.
+
+### Prepare User Data
```

Comment:
This header appears on the right sidebar, but will not work when clicked if the **Manual File Creation** tab is not active. This is a limitation with Docusaurus, but are we okay with this?